src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Log/FunctionId.cs

```diff
@@ -482,5 +482,7 @@ internal enum FunctionId
 
         Renamer_RenameSymbolAsync = 386,
         Renamer_FindRenameLocationsAsync = 386,
+
+        ChangeSignature_Data = 387,
```

Comment:
@CyrusNajmabadi I've merged this as-is, with my number adjacent to your duplicated numbers. You'll either have to pick 386 & 388 to dodge mine, or you can change my number too as long as it's before Preview 1 ships. /cc: @jinujoseph